src/commands/validate-tx.cpp

```diff
@@ -51,23 +51,14 @@ console_result validate_tx::invoke(std::ostream& output,
 
     callback_state state(error, output);
 
-    auto on_done = [&state](const chain::point::indexes& indexes)
+    auto on_done = [&state](const code& error)
     {
-        if (indexes.empty())
-        {
-            state.output(BX_VALIDATE_TX_VALID);
-            return;
-        }
-
-        const auto unconfirmed = join(numbers_to_strings(indexes), ", ");
-        state.output(format(BX_VALIDATE_TX_UNCONFIRMED_INPUTS) % unconfirmed);
+        state.succeeded(error);
+        state.output(BX_VALIDATE_TX_VALID);
```

Comment:
If the error is anything other than success, on_error is called (as per decode_reply in client).  So on_done should always succeed with a success code. no?  As-is, I get this output:

```
featre not implemented
```

The typo should probably be corrected in libbitcoin.